src/__tests__/fire-event.test.tsx

```diff
@@ -9,6 +9,7 @@ import {
   View,
 } from 'react-native';
 import { fireEvent, render, screen } from '..';
+import '../matchers/extend-expect';
```

Comment:
It should be possible to import that file in a jest setup file so that matchers are available globally no?